server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/model/MailboxFactory.scala

```diff
@@ -31,12 +32,64 @@ import reactor.core.scala.publisher.SMono
 import scala.jdk.CollectionConverters._
 import scala.jdk.OptionConverters._
 
+object MailboxValidation {
+  private val errors: Seq[String] = Seq()
```

Comment:
I really don't think this is good... I have other ideas, but would like people's insight on managing the errors for the mailbox validation